src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public enum Color {
+    BLACK,
+    WHITE,
+    VIOLET,
+    PURPLE,
+    BLUE,
+    GREEN,
+    RED;
+
+    @Override
+    public String toString() {
+        return "default";
+    }

```

Comment:
not required by the task
```suggestion
    RED
```

Reply:
Ok, thanks